docs/reference/cluster_manifest.md

```diff
@@ -359,3 +363,28 @@ CPU and memory limits for the sidecar container.
 * **memory**
   memory limits for the sidecar container. Optional, overrides the
   `default_memory_limits` operator configuration parameter. Optional.
+
+## Custom TLS certificates
+
+By default, the spilo image generates its own TLS certificate during startup.
+This certificate is not secure since it cannot be verified and thus doesn't
+protect from active MITM attacks.
+
+By setting the `secretName` value in this section, the cluster will switch to
+load the given Kubernetes Secret into the container as a volume and uses that
+as the certificate instead. It is up to the user to create and manage the
+Kubernetes Secret either by hand or using a tool like the CertManager
+operator.
+
+When `secretName` is set, the `spiloFSGroup` value will also be set to `103`
+by default unless defined otherwise by the user. This is necessary so that the
+postgres instance can read from the secret volume.
```

Comment:
This redundant to the user docs and can be removed I think.

Reply:
Up to the title?

Reply:
No just the section I've marked (lines 379-381)